src/app/carbon-estimator-form/carbon-estimator-form.component.ts

```diff
@@ -20,6 +21,26 @@ const locationDescriptions: Record<WorldLocation, string> = {
   'LATIN AMERICA AND CARIBBEAN': 'in Latin America or the Caribbean',
 };
 
+export type ValidationError = {
+  inputId: string;
+  errorMessage: string;
+};
+
+const errorConfig = {
```

Comment:
Maybe put this in the `.constants.ts` file?

Would also feel better to have the ValidationError in a separate file than having a circular reference.

Reply:
Sounds good. Should I move the `locationDescriptions` to the `.constants.ts` file as well?

Reply:
Yeah, that seems like a good idea. I'd forgotten there were two things with the same name in the same file.